pkg/grpc/server.go

```diff
@@ -314,7 +314,9 @@ func shouldRenewCert(certExpiryDate time.Time, certDuration time.Duration) bool
 
 func (s *server) getGRPCAPILoggingInfo() grpc_go.UnaryServerInterceptor {
 	return func(ctx context.Context, req interface{}, info *grpc_go.UnaryServerInfo, handler grpc_go.UnaryHandler) (interface{}, error) {
-		s.infoLogger.Info("gRPC API Called: ", *info)
+		if info != nil {
```

Comment:
Needs explanation on why this is part of the hotfix. I see this was labeled as `P1` which I agree with. We don't include commits normally for `P1` issues in a patch release.

Reply:
According to me, it is breaking the functionality(enable-api-logging) that we added in 1.7. So it needs to go in to 1.7.

Reply:
Thanks for explaining @amulyavarote, we'd like to have maintainer consensus (2/3 agreeing in this case) before merging. I'll discuss with @artursouza and/or @daixiang0 to determine this.

Reply:
Sure. Thanks @yaron2

Reply:
As discussed offline, @yaron2 and I agreed on this being merged.

Reply:
I am creating a new PR containing the doc change.